backend/src/Designer/Services/Implementation/TextsService.cs

```diff
@@ -305,25 +305,27 @@ public async Task<string> UpdateKey(string org, string repo, string developer, I
         }
 
         /// <inheritdoc />
-        public async Task UpdateRelatedFiles(string org, string app, string developer, List<TextIdMutation> keyMutations)
+        public async Task<List<string>> UpdateRelatedFiles(string org, string app, string developer, List<TextIdMutation> keyMutations)
```

Comment:
I also think it is preferable to use the repository.SaveLayout directly from the handler such as done for the other handlers instead of using the service.

So the absolute best way to solve this would be to delete this method from the service and write the logic inside the handler in the similar way as done with the `ComponentIdChangedLayoutsHandler` 🫣

Reply:
I do see your point here, and i agree. But i think this could be better to move to another issue/PR as you touched on in the review summary comment.